content/src/service/pointers/PointerManager.ts

```diff
@@ -12,72 +9,38 @@ export class PointerManager {
    * Commit a new entity, and return the 'before & after' generated by the deployment
    */
   async referenceEntityFromPointers(
```

Comment:
Can we add some more comments here explaining what does this method do in general and in each line?

Reply:
Does this endpoint make an update in the db or is just a read only?

Reply:
It's read-only